docs/content/sips/005-admin-api.md

```diff
@@ -0,0 +1,92 @@
+title = "SIP 005 - The Admin API
```

Comment:
Minor thing, but the naming convention for SIPs in draft phase are SIP XXX - <Title>. Once merged, you can name it to its concrete index.